react-app/src/components/RemoveClientModal/RemoveClientModal.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+
+const style = {
+    position: 'absolute' as 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 500,
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 4,
+};
+
+export default function RemoveClientModal( {deleteClients}: any ) {
+    const [open, setOpen] = React.useState(false);
+    const handleOpen = () => setOpen(true);
+    const handleClose = () => setOpen(false);
+    const handleYes = () => { deleteClients(); setOpen(false); }
+
+    return (
+        <div>
+            <Button variant="contained" onClick={handleOpen}>
+                Delete Selected Teams
```

Comment:
Might make more sense to have this button say 'Delete Selected Clients'

Reply:
Good catch! Fixed.